core/lib/default_da_clients/src/object_store/config.rs

```diff
@@ -0,0 +1,14 @@
+use zksync_config::ObjectStoreConfig;
+use zksync_env_config::envy_load;
+
+pub struct ObjectStoreDAConfig {
+    pub config: ObjectStoreConfig,
+}
```

Comment:
Nit: IMO, it would make sense to follow the format of other object store configs:

```rust
pub struct ObjectStoreDAConfig(pub ObjectStoreConfig);
```

Also, please derive `Debug` for public types.